### ISSUE TITLE
Add Google Analytics code to portal

### ISSUE THREAD

**Issue by kyyberi** [PlatformOfTrust/pot-websites#102]
Add google analytics tracking to Developer portal so that it is use in every page. 
Implementation tactics free: use Gatsby modelu or hardcore js injection. 

```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-121413138-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-121413138-1');
</script>
```

## Acceptance criteria:

- [ ] Tracking covers all pages in Developer Portal
- [ ] Analytics view in Google shows initial data from the tracking.

**Comment by MikaHamalainen** [comment]
https://www.gatsbyjs.org/docs/adding-analytics/

**Comment by kyyberi** [comment]
Implemented in PR Branch Quick fixes #118
